### PR TITLE
solving #209 (Languages not alphabetically sorted)

### PR DESCRIPTION
**Pull request by mahinourElsheikh**
A quick description of the changes made in this pull request. Make sure to reference the related issue (if there is one).

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 window.Vue = require("vue")
-const { allLanguages, topLanguages: toplangs } = require("./languages")
+const { allLanguages } = require("./languages")
```
Comment (damcclean) [inline src/js/app.js:2]: If we're no longer using the `topLanguages` array, can we remove it from the `languages.js` file please?

**Review by damcclean — CHANGES_REQUESTED** [review]
Looks good, thanks for your pull request!

I've left a comment below, hopefully you can get to it.

File: src/js/app.js (hunk 1 above)
Comment (mahinourElsheikh) [reply src/js/app.js:2]: okay sure

**Comment by damcclean** [comment]
Thanks for the changes! Have a nice day!

**Pull request merged**